enderio-base/src/main/java/com/enderio/base/common/block/EIOBlockEntity.java

```diff
@@ -37,7 +37,7 @@ public void onLoad() {
         super.onLoad();
 
         if (supportsRedstonePower()) {
-            updateRedstonePower();
+            updateRedstonePower(true);
```

Comment:
I proposed this in Discord but I'll leave the comment here too :)

I'd personally rather call onRedstonePowerChanged after this call -instead- of adding a new boolean parameter to updateRedstonePower. Booleans have little meaning when reading through the code, so if we instead call the changed method and explain why we do it that may make more sense :)

Reply:
Done